docs/guides/jsdom.md

```diff
@@ -11,6 +11,31 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
+```js
+/* setup.js */
+
+var { JSDOM } = require('jsdom');
+var jsdom = new JSDOM('');
```

Comment:
I don't think this is enough, either. With the proposed doc changes, I see the same errors as reported here: https://github.com/tmpvar/jsdom/issues/1827.